### PR TITLE
Commerce 7162

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-7162

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/edit_cp_definition_diagram_setting.js [hunk 1]
```diff
@@ -9,15 +9,29 @@
...
+			DIAGRAMS_ENDPOINT + diagramId,
+			themeDisplay.getPortalURL()
+		);
+		fetch(url, {
+			body: JSON.stringify({
+				type: type.value,
+			}),
+			headers: HEADERS,
+			method: 'PATCH',
+		});
```
Comment (FabioDiegoMastrorilli) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/edit_cp_definition_diagram_setting.js:34]: We should handle errors via `catch` and `!response.ok` so the user can be aware that something is not ok.

(btw I don't see it as blocker in this case)

**Comment by andrea-ale-sbarra** [comment]
ci:forward

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/edit_cp_definition_diagram_setting.js (hunk 1 above)
Comment (andrea-ale-sbarra) [reply modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/edit_cp_definition_diagram_setting.js:34]: thank you Fabio, will do this after COMMERCE-5254 is merged so I can move this logic in the utilities. js file

**Pull request closed without merging**